README.md

```diff
@@ -66,3 +66,77 @@ To support browsers that don't have built-in support for the Web Components API,
 import "@webcomponents/custom-elements";
 import "@mercurymedia/elm-ag-grid";
 ```
+
+## Custom views for cells
+
+Custom cell content (such as a button or link) can be created and used as a separate Elm application. This application must be defined as `Browser.element` in load it into the table cell.
```

Comment:
Not as a separate elm application, but a separate elm app can be used as a renderer for cells.

Reply:
I changed the sentences a bit. Should sound more accurate now.